### PR TITLE
DOC: fix PR09,PR08 docstring errors in pandas.plotting

### LINKED ISSUE #28687: DOC: Docstrings Formatting Error PR09,PR08 in pandas.plotting
fix PR09,PR08 errors in `pandas.plotting`: 

```
pandas.plotting.andrews_curves: Parameter "frame" description should finish with "."
pandas.plotting.andrews_curves: Parameter "color" description should finish with "."
pandas.plotting.andrews_curves: Parameter "kwds" description should finish with "."
pandas.plotting.autocorrelation_plot: Parameter "kwds" description should finish with "."
pandas.plotting.parallel_coordinates: Parameter "class_column" description should finish with "."
pandas.plotting.parallel_coordinates: Parameter "cols" description should finish with "."
pandas.plotting.parallel_coordinates: Parameter "ax" description should start with a capital letter
pandas.plotting.parallel_coordinates: Parameter "ax" description should finish with "."
pandas.plotting.parallel_coordinates: Parameter "color" description should finish with "."
pandas.plotting.parallel_coordinates: Parameter "use_columns" description should finish with "."
pandas.plotting.parallel_coordinates: Parameter "xticks" description should finish with "."
pandas.plotting.parallel_coordinates: Parameter "axvlines" description should finish with "."
pandas.plotting.parallel_coordinates: Parameter "axvlines_kwds" description should finish with "."
pandas.plotting.parallel_coordinates: Parameter "sort_labels" description should finish with "."
pandas.plotting.parallel_coordinates: Parameter "kwds" description should finish with "."
pandas.plotting.scatter_matrix: Parameter "alpha" description should start with a capital letter
pandas.plotting.scatter_matrix: Parameter "alpha" description should finish with "."
pandas.plotting.scatter_matrix: Parameter "figsize" description should start with a capital letter
pandas.plotting.scatter_matrix: Parameter "figsize" description should finish with "."
pandas.plotting.scatter_matrix: Parameter "grid" description should start with a capital letter
pandas.plotting.scatter_matrix: Parameter "grid" description should finish with "."
pandas.plotting.scatter_matrix: Parameter "diagonal" description should start with a capital letter
pandas.plotting.scatter_matrix: Parameter "diagonal" description should finish with "."
pandas.plotting.scatter_matrix: Parameter "marker" description should finish with "."
pandas.plotting.scatter_matrix: Parameter "hist_kwds" description should finish with "."
pandas.plotting.scatter_matrix: Parameter "density_kwds" description should finish with "."
pandas.plotting.scatter_matrix: Parameter "range_padding" description should start with a capital letter
pandas.plotting.scatter_matrix: Parameter "range_padding" description should finish with "."
pandas.plotting.scatter_matrix: Parameter "kwds" description should finish with "."
pandas.plotting.table: Parameter "data" description should start with a capital letter
pandas.plotting.table: Parameter "data" description should finish with "."
pandas.plotting.table: Parameter "kwargs" description should start with a capital letter
```

This is a subset of #28602 and #27977

### PR DESCRIPTION
**Pull request by josibake**
fixes the errors along with minor changes to standardize kwds -> **kwargs
closes #28687

verified this fixes PR09,PR08 errors with:

```
./scripts/validate_docstrings.py --errors=PR09,PR08 | grep "pandas.plotting"
```

- [x] closes #xxxx
- [x] tests added / passed

### CODE REVIEW

**Comment by josibake** [comment]
this is related to #27977 and #28602 (adding for visibility)

File: pandas/core/frame.py [hunk 1]
```diff
@@ -7939,8 +7947,8 @@ def idxmin(self, axis=0, skipna=True):
         Parameters
         ----------
         axis : {0 or 'index', 1 or 'columns'}, default 0
-            0 or 'index' for row-wise, 1 or 'columns' for column-wise
-        skipna : bool, default True
+            The axis to use. 0 or 'index' for row-wise, 1 or 'columns' for column-wise
+        skipna : boolean, default True
```
Comment (WillAyd) [inline pandas/core/frame.py]: I think this should stay `bool` - surprised it doesn't error on validation

**Review by WillAyd — CHANGES_REQUESTED** [review]

File: pandas/core/frame.py (hunk 1 above)
Comment (josibake) [reply pandas/core/frame.py]: I'm actually surprised to see this in here at all, as I was only editing the `_misc.py` file? still figuring out how to merge fork/merge/etc correctly

Comment (WillAyd) [reply pandas/core/frame.py]:
```sh
git fetch upstream
git merge upstream/master
# fix merge conflicts, if any then optionally
# git merge continue
```

Comment (josibake) [reply pandas/core/frame.py]: got it, thanks. i think also having my local branch have the upstream as `upstream/master` was causing problems.

**Review by WillAyd — APPROVED** [review]

**Comment by WillAyd** [comment]
Thanks @josibake !

**Pull request merged**